data/json/npcs/classes.json

```diff
@@ -106,7 +106,8 @@
       { "skill": "ALL", "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "rng": [ 0, -4 ] } ] } ] } },
       { "skill": "gun", "bonus": { "rng": [ 1, 3 ] } },
       { "skill": "pistol", "bonus": { "rng": [ 1, 3 ] } },
-      { "skill": "rifle", "bonus": { "rng": [ 0, 2 ] } }
+      { "skill": "rifle", "bonus": { "rng": [ 0, 2 ] } },
+	  { "skill": "survival", "bonus": { "rng": [ 0, 2 ] } }
```

Comment:
This is fine, but I wouldn't muck around too much with skills and things... This is one of the classes that can show up as a random npc so it has a wide impact.

That said it's reasonable for them to get a bit more chance of having survival.

Reply:
My thinking. Someone who lived on a farm/ranch would likely have some knowledge, and it's not like this tiny boost makes them crazy. Mostly for flavour.